### ISSUE TITLE
Problems implementing API endpoints or Graphql to work in prod and dev at the same time

### ISSUE THREAD

**Issue by pardejini** [faceyspacey/redux-first-router-demo#21]
Hi there, I've  added a simple api and graphql to this project but I'm constantly having issues.
Every time I try to import a file to server/index.js webpack seems to disagree with me. Sometimes it works in dev mode other times in prod, but never both at the same time. Any tips on how to implement api routes or graphql to this project or even to the base redux-first-router-demo boilerplate?

**Comment by faceyspacey** [comment]
split the DEV and production code into 2 separate files if you can get individual modes to work. and then just create a script for each.

u shouldnt have any issues with an API though. any code that requires webpack should be required/imported from `server/render.js`. thats the webpack entry point. so if you try to require things like CSS modules into the `server/index.js` entry point it will not work. 

basically there's no reason webpack should be disagreeing with you for importing into `server/index.js` unless are are bundling that file with webpack OR trying to use stuff that requires webpack loaders/plugins in that file (i.e. imported to that file).
